README.md

```diff
@@ -48,7 +52,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+# # belle = town [;residents].delete
+# town: [:castle][:guest].push (belle)
```

Comment:
This is close.  The punctuation isn't correct.  Please review out suggestion in the solution branch.